semgrep_output_v1.atd

```diff
@@ -868,6 +868,124 @@ type cli_output_extra = {
     ~skipped_rules: skipped_rule list;
 }
 
+(*****************************************************************************)
+(* Matching diagnosis *)
+(*****************************************************************************)
+
```

Comment:
please add a big EXPERIMENTAL comment, so people know they can't rely on it, and it could change.